modules/gitops-argo-cd-properties.adoc

```diff
@@ -59,37 +72,56 @@ a|* `<enabled>` - Toggle high availability support globally for Argo CD.
 
 |`image`    |The container image for all Argo CD components. This overrides the `ARGOCD_IMAGE` environment variable.|`argoproj/argocd` |
 
+|`import`    |Import configuration options for Argo CD.| `__<object>__`
+a|* `<name>` - The name of an `ArgoCDExport` resource from which data can be imported.
+  * `<namespace>` - The namespace for the `ArgoCDExport` resource referenced by `name` field. If this field is not set, the namespace of `ArgoCDExport` resource is set to the same namespace as ArgoCD by default.
+
 |`ingress`    |Ingress configuration options.| `__<object>__` |
 
 |`initialRepositories`    |Initial Git repositories to configure Argo CD to use upon creation of the cluster.|`__<empty>__` |
 
-|`notifications`    |Notifications controller configuration options.|`__<object>__`
-a|* `<enabled>` - The toggle to start the notifications-controller.
+|`initialSSHKnownHosts`    |Defines initial SSH Known Hosts data for Argo CD to use upon creation of the cluster for connecting Git repositories by SSH.| `__<default_Argo_CD_Known_Hosts>__`
+a|* `<Excludedefaulthosts>` - Indicates whether you want to add the default list of SSH Known Hosts provided by Argo CD.
+  * `<keys>` - Describes a custom set of SSH Known Hosts that you want to incorporate into your Argo CD server.
+
+|`kustomizeBuildOptions`    |The build options and parameters to use with `kustomize build`.|`__<empty>__` |
+
+|`kustomizeVersions`    |Defines a list of `Kustomize` versions that have been configured in the Argo CD repo server container image.|`__<empty>__`
+a|* `<path>` - The path of the `Kustomize` version in the file system of the Argo CD repo server container image.
+  * `<version>` - The `Kustomize` version in the `vX.Y.Z` format configured in the Argo CD repo server container image.
+
+|`monitoring`    |Defines the workload status monitoring configuration for your instance.| `__<object>__`
+a|* `<disableMetrics>` - Configure this field to enable or disable the collection of metrics for your instance.
+  * `<enabled>` - Indicates whether the workload status monitoring is enabled for your instance.
+
+|`notifications`    |Notifications Controller configuration options.|`__<object>__`
+a|* `<enabled>` - The toggle to start the Notifications Controller.
+  * `<env>` -  Environment to set for the Notifications Controller workloads.
   * `<image>` - The container image for all Argo CD components. This overrides the `ARGOCD_IMAGE` environment variable.
   * `<version>` - The tag to use with the Notifications container image.
+  * `<replicas>` - The number of replicas to be executed for the Notifications Controller.
```

Comment:
```suggestion
  * `<replicas>` - The number of replicas to be run for the Notifications Controller.
```
Per IBM Style, use _execute_ with caution, only if a simpler term, such as _run_, _issue_, _start_, or _enter_ is inappropriate.

Reply:
Fixed.